ideals.html

```diff
@@ -151,6 +151,12 @@
 	<p>Cockpit allows others to add additional <span class="caps">UI</span> pieces. We do
 	not allow arbitrary extension of the <span class="caps">UI</span> all over the place, but
 	at distinct extension&nbsp;points.</p>
+
+	<h2>Cockpit is&nbsp;tested</h2>
+	<p>Every feature and change in Cockpit is covered by an automatic test.
```

Comment:
Do we meet those strict criteria? "Cockpit features and changes are covered by automatic tests."?